src/platform/middleware.ts

```diff
@@ -73,3 +75,48 @@ export const gracefullyDenyDeletedDevices: RequestHandler = (
 			next();
 		});
 };
+
+// register the device state change handler which updates the device model...
+_.once(() => {
```

Comment:
You can ditch this `_.once` altogether and just run the code at module level

Reply:
So if I just call the function directly, then when the `middleware.ts` is imported it will be run... what happens if it's imported twice in different locations? will it be evaluated twice and therefore add 2 event handlers?

Reply:
Nope, import never evaluates a file twice.